B827EBFFFEDC465B

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE00F9D7",
```

Comment:
This EUI must match the one in the file name, and the file name must include the `.json` extension.